docs/src/modules/reference/pages/release-notes.adoc

```diff
@@ -28,6 +28,15 @@ xref:java:consuming-producing.adoc#s2s-eventing[Service to service eventing] is
 xref:concepts:multi-region.adoc[Request-region primary selection mode]: The primary region changes when another region receives a write request. Upon a write request to an entity in a region that is not the primary it will move its primary. The new primary ensures that all preceding events from the previous primary have been fully replicated and applied (i.e. persisted) before writing the new event, and thereby guarantees strong consistency when switching from one region to another. Subsequent write requests to the primary region of the entity are handled locally without any further coordination. Write requests to other regions will trigger the same switch-over process. All other entity instances operate unimpeded during the switch-over process.
 
 ****
+* Akka Runtime 1.4.3
+    - automatic deduplication of views updates
```

Comment:
I think we can add another item for the deduplication best practices documentation

Reply:
You mean on the section above with the hightlights right? wondered the same about mentioned this deduplication feature there with some more details and then linking to the new "deduplication best practices documentation". could be good. let me update.

Reply:
See what you think now.